.devcontainer/.devcontainer.json

```diff
@@ -1,10 +1,10 @@
 {
     "name": "Couchbase [Framework] Workspace",
```

Comment:
Should this be `Couchbase Node.js Workspace`?

Reply:
Yes, good catch!